### PR TITLE
Updated pipeline with new btrack loc. features and separate multiclass notebooks

### PR DESCRIPTION
**Pull request by nthndy**
#9 clarified this
Also changed around the phrasing of the generator function in classifier step as even though btrack functionality is changed to accept dask, the generator approach is much much quicker (maybe want to clarify this further in some doc?)

### CODE REVIEW

**Comment by nthndy** [comment]
will wait until we merge the octopuslite-reader pr until i merge this one

**Comment by nthndy** [comment]
this will need the btrack class id changes to go ahead before merging

**Review by quantumjot — APPROVED** [review]
I don't have a huge amount of time so can't go through this very carefully. It looks fine overall (the main thing is that it works for you!). A few notes though:
* In general, it's better to make small isues and then each PR is a small change, rather than thousands of changes like this one. It makes it easier to review
* I don't really understand why you're using two separate H5 files for the tracking in the multiclass case - the whole point of the `obj_typ_1` etc... is that they can all be stored in the same file.

I will merge and you can make any further updates with follow-up PRs.

Hope that helps!

**Pull request merged**
